src/snowflake/cli/api/commands/flags.py

```diff
@@ -134,11 +134,12 @@ def _password_callback(value: str):
 
 PrivateKeyPathOption = typer.Option(
     None,
+    "--private-key-file",
     "--private-key-path",
-    help="Snowflake private key path. Overrides the value specified for the connection.",
+    help="Snowflake private key file path. Overrides the value specified for the connection.",
     hide_input=True,
     callback=_callback(
```

Comment:
Should we add deprecation info in callback?

Reply:
We don't know which flag was used `"--private-key-path"` is an alias. I tried to add context to callback function but failed.